src/App.js

```diff
@@ -1,54 +1,28 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import { Provider } from 'react-redux'
+
 import { Router } from 'react-router-dom'
+
 import history from './utils/history'
-import store from './store'
-import './styles/styles.css'
-import AppWithD2ContextAndTheme from './components/AppWithD2ContextAndTheme'
 import SectionLoader from './components/SectionLoader'
 import SnackbarContainer from './components/SnackbarContainer'
 import DialogContainer from './components/DialogContainer'
 import SharingDialogContainer from './components/SharingDialogContainer'
 
-import { DataProvider } from '@dhis2/app-runtime'
-import { HeaderBar } from '@dhis2/ui-widgets'
-import { CssReset } from '@dhis2/ui-core'
-
-import i18n from '@dhis2/d2-i18n'
-
-import 'typeface-roboto'
-import 'material-design-icons-iconfont/dist/material-design-icons.css'
-
 /**
  * Main Component. Renders a AppWithD2ContextAndTheme wrapped in a Provider containing the HeaderBar,
```

Comment:
This is no longer true right?

Reply:
Good find, I updated the comment. I wish I would have never added these jsDocs to this app. They just get out of sync with the actual code so easily 😞